client/js/lounge.js

```diff
@@ -729,6 +729,7 @@ $(function() {
 		var self = $(this);
 		viewport.toggleClass(self.attr("class"));
 		e.stopPropagation();
+		chat.find(".chan.active .chat").trigger("msg.sticky");
```

Comment:
Hmm yes, but that's not what I meant. Probably explained myself poorly.

Let's put it that way: go to a channel, scroll all the way up, then click on the user list button. According to this change, I'm reading that this should force the window to go stickied to bottom, even though you probably don't want that if you weren't at the bottom before clicking. But this behavior doesn't happen, user stays at the top. This is great, you don't want to get moved to bottom if you weren't before, but I don't understand how that's possible.